docs/source/cli/model-training-inference/configuration-run.rst

```diff
@@ -19,19 +17,26 @@ GraphStorm's model training and inference CLIs (both task-specific and task-agno
 - **ssh-port**: SSH port used by the host node to communicate with the other nodes in the cluster.
 - **ssh-username**: Optional. When issuing commands (via ssh) to cluster, use the provided username in the ssh command.
 - **graph-format**: The format of the graph structure of each partition. The allowed formats are csr, csc and coo. A user can specify multiple formats, separated by ",". For example, the graph format is "csr,csc".
-- **cf** or **yaml-config-file**: (**Required**) Path to the YAML configuration file.
 - **extra-envs**: Extra environment parameters need to be set. For example, you can set the LD_LIBRARY_PATH and NCCL_DEBUG by adding:
 
     - --extra_envs LD_LIBRARY_PATH=/usr/local/cuda/lib64:$LD_LIBRARY_PATH
     - --extra-envs LD_LIBRARY_PATH=/usr/local/cuda/lib64:$LD_LIBRARY_PATH
     - NCCL_DEBUG=INFO
 - **do-nid-remap**: Do GraphStorm node ID to Raw input node ID remapping for prediction results and node embeddings. Default is True.
 
+Model Training and Inference Configurations
+--------------------------------------------
+GraphStorm provides dozens of configurable parameters for users to control their training and inference tasks. You can use a yaml config file to define these parameters or you can use command line arguments to define and update these parameters. Specifically, GraphStorm parses yaml config file first. Then it parses arguments to overwrite parameters defined in the yaml file or add new parameters.
+
+Yaml File Configurations
+.........................
+- **cf** or **yaml-config-file**: (**Required**) Path to the YAML configuration file.
+
 ..  note:: Below configurations can be set either in a YAML configuration file or be added as arguments of launch command.
 
 Environment Configurations
-----------------------------
-- **backend**: (**Required**) PyTorch distributed backend, the suggested backend is gloo. Support backends include gloo and nccl
+...........................
+- **backend**: (**Required**) PyTorch distributed backend, the suggested backend is `gloo`. Support backends include `gloo` and `nccl`.
```

Comment:
I just double check it here, the backend actually is not required, we return gloo as default. https://github.com/awslabs/graphstorm/blob/60edf9b907f5d0a5a29745c9bf43daa5c190798a/python/graphstorm/config/argument.py#L837

Reply:
removed.